testing/endtoend/evaluators/beaconapi_evaluators/beacon_api.go

```diff
@@ -24,612 +23,419 @@ import (
 )
 
 var (
-	errSszCast             = errors.New("ssz response is not a byte array")
-	errJsonCast            = errors.New("json response has wrong structure")
-	errEmptyPrysmData      = errors.New("prysm data is empty")
-	errEmptyLighthouseData = errors.New("lighthouse data is empty")
+	errSszCast   = errors.New("SSZ response is not a byte array")
+	errEmptyData = errors.New("data is empty")
 )
 
-type metadata struct {
-	start            primitives.Epoch
-	basepath         string
-	params           func(encoding string, currentEpoch primitives.Epoch) []string
-	requestObject    interface{}
-	prysmResps       map[string]interface{}
-	lighthouseResps  map[string]interface{}
-	customEvaluation func(interface{}, interface{}) error
+const (
+	msgWrongJson = "JSON response has wrong structure, expected %T, got %T"
+)
+
+type meta interface {
```

Comment:
should we put this in a separate file? this and metadata ( wondering if metadata is still the right name for this...

Reply:
Good point, it should be renamed. I can put it in a separate file too.